src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -162,7 +163,8 @@ const peerDisconnectedChangeHandler = async (
 ) => {
   if (connectedMeerKat === event.payload.dAppAddress) {
     dispatch(setConnectedWallet(null));
-    dispatch(setToastMsg(ToastMsgType.DISCONNECT_WALLET_SUCCESS));
+    // Sometime disconnect event fired twice and make UI display 2 toast message with same content
```

Comment:
Thanks for reporting this. We can remove this comment as I've created a ticket for investigating that.

https://cardanofoundation.atlassian.net/browse/DTIS-1342

In the future it's better if we just open a ticket rather than leaving the comment here, otherwise we may forget about it.

Reply:
Done